utils/documentation_tests.txt

```diff
@@ -92,7 +92,6 @@ src/transformers/models/glpn/modeling_glpn.py
 src/transformers/models/gpt2/configuration_gpt2.py
 src/transformers/models/gpt2/modeling_gpt2.py
 src/transformers/models/gptj/modeling_gptj.py
-src/transformers/models/gptsan_japanese/modeling_gptsan_japanese.py
```

Comment:
I know you mentioned that we should remove an entry if there is no smaller checkpoint. (But if just a simple usage of FP16 works, should I apply it? I haven't tried though.)

Reply:
I don't think the model checkpoint is usable as is anyway since it contains a index for a sharded checkpoint but no shards. But yes, like for gpt_neox_japanese, there is no point testing a 6.7B parameter checkpoint.